driver-core/src/main/com/mongodb/connection/ConnectionPoolSettings.java

```diff
@@ -343,6 +366,21 @@ public long getMaintenanceFrequency(final TimeUnit timeUnit) {
         return connectionPoolListeners;
     }
 
+    /**
+     * The maximum number of connections a pool may be establishing concurrently.
+     * Establishment of a connection is a part of its life cycle
+     * starting after a {@link ConnectionCreatedEvent} and ending before a {@link ConnectionReadyEvent}.
+     * <p>
+     * Default is 2.
+     *
```

Comment:
Nit: No close p tag is required?  IntelliJ always add it for me.

Reply:
I guess, I have never given it proper thought. [Examples posted by Oracle](https://www.oracle.com/ca-en/technical-resources/articles/java/javadoc-tool.html#format) do not close the tag, so I blindly followed. This is similar to the situation we discussed yesterday about examples violating best practices because "it's simpler this way".

- The standard doclet specification states [_"HTML constructs should be written in HTML 5"_](https://docs.oracle.com/en/java/javase/17/docs/specs/javadoc/doc-comment-spec.html#html-content)
- HTML 5 states [_"A p element's end tag can be omitted if the p element is immediately followed by an address, article, aside, blockquote, details, div, dl, fieldset, figcaption, figure, footer, form, h1, h2, h3, h4, h5, h6, header, hgroup, hr, main, menu, nav, ol, p, pre, section, table, or ul element, or if there is no more content in the parent element and the parent element is an HTML element that is not an a, audio, del, ins, map, noscript, or video element, or an autonomous custom element."_](https://html.spec.whatwg.org/multipage/grouping-content.html#the-p-element).

I don't think a human can reasonably apply the above omission rule, and since omitting the end tag for the `p` element is optional, we should just not do it. Thank you for pointing this out, I will stop omitting the end tag, I also added closing tags in the PR.